### PR TITLE
XIVY-12055 Completed Implementation: SRF Weather Connector, Demo Project, Process Test, WebTest and Product Descriptions

### PR DESCRIPTION
**Pull request by ivy-edp**

### CODE REVIEW

**Comment by alexsuter** [comment]
can be reviewed by Lukas :). I will review the connector of Giger.

File: srf-weather-connector/processes/srfWeatherForecast.p.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "call(Number,String)",
+      "config" : {
+        "callSignature" : "call",
+        "input" : {
+          "params" : [
+            { "name" : "zip", "type" : "Number" },
+            { "name" : "locationName", "type" : "String" }
```
Comment (ivy-lli) [inline srf-weather-connector/processes/srfWeatherForecast.p.json:17]: As you already marked your call sub start as a connector, it now appears in the connector palette -> very nice!
You can help adopters of your connector even more if you add descriptions to this start and its parameters.

File: srf-weather-connector/processes/srfWeatherForecast.p.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      "id" : "f7",
+      "type" : "Alternative",
+      "name" : "is geolocationID empty?",
+      "visual" : {
+        "at" : { "x" : 336, "y" : 64 },
+        "labelOffset" : { "x" : 14, "y" : 34 }
+      },
+      "connect" : [
+        { "id" : "f6", "to" : "f5", "condition" : "in.location.geolocationId != \"\"" },
+        { "id" : "f11", "to" : "f10", "via" : [ { "x" : 336, "y" : 144 } ], "condition" : "in.location.geolocationId == \"\"" }
```
Comment (ivy-lli) [inline srf-weather-connector/processes/srfWeatherForecast.p.json:122]: I think you only have to define the first condition, the second is then the "else" case.

File: srf-weather-connector/webContent/icons/sun(1).jpg
Comment (ivy-lli) [inline srf-weather-connector/webContent/icons/sun(1).jpg:1]: is this image used?

File: srf-weather-connector/webContent/icons/sunrise.png
Comment (ivy-lli) [inline srf-weather-connector/webContent/icons/sunrise.png:1]: is this image used?

File: srf-weather-connector/webContent/layouts/basic-10.xhtml
Comment (ivy-lli) [inline srf-weather-connector/webContent/layouts/basic-10.xhtml:1]: I think you don't have a html dialog in the connector right? So you can remove the webContent folder.

**Review by ivy-lli — CHANGES_REQUESTED** [review]

**Pull request merged**